cernopendata/config.py

```diff
@@ -118,6 +118,12 @@
         view_imp='cernopendata.modules.records.utils:file_download_ui',
         record_class='invenio_records_files.api:Record',
     ),
+    recid_files_root=dict(
+        pid_type='recid',
+        route='/record/<pid_value>/files/root',
```

Comment:
This is not only for `root` files, so we may invent a different name to express the notion of "asset files behind index files".